tests/acceptance/test_examples.py

```diff
@@ -15,13 +15,12 @@
 
 
 @pytest.mark.parametrize("example_path", EXAMPLE_PATHS)
-def test___shipping_example___run___no_errors(example_path: Path):
-    local_system = nidaqmx.system.System.local()
+def test___shipping_example___run___no_errors(example_path: Path, system):
```

Comment:
This test runs the example programs, which do not support gRPC, so mark it library_only and add a comment stating the reason.